### PR TITLE
Is 56 home page top

### PR DESCRIPTION
**Pull request by shriyan44**

### CODE REVIEW

File: frontend/isoko/src/components/Home/LocationSearchBar.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            IndexName: 'Isoko-Index',
+            Text: value,
+            MaxResults: 7,
+            FilterCountries: ['USA'],
+         })
+         .promise();
+
+      const options = rsp.Results.map((value) => value.Text);
+      setOptionsState(options);
+      console.log(optionsState);
```
Comment (jpoist97) [inline frontend/isoko/src/components/Home/LocationSearchBar.tsx:58]:
```suggestion
```

File: frontend/isoko/src/components/Home/LocationSearchBar.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+   const credentials = new AWS.CognitoIdentityCredentials({
+      IdentityPoolId: 'us-west-2:7e6f6851-3cee-4edf-af12-50c3e00f365b',
+   });
+
+   const client = new AWS.Location({
+      credentials,
+      region: 'us-west-2', // region containing the identity pool
+   });
+
+   const [optionsState, setOptionsState] = useState<Array<string>>([]);
```
Comment (jpoist97) [inline frontend/isoko/src/components/Home/LocationSearchBar.tsx:43]: nit: can you move this to line 34, usually hooks should be the first thing in a function component

File: frontend/isoko/src/pages/Home.tsx [hunk 3]
```diff
@@ -1,7 +1,92 @@
...
+   z-index: -1;
+   // padding: 1em;
+   background-image: url(${sculptures});
+   background-size: cover;
+   background-position: center;
+   background-repeat: no-repeat;
+   // @media ${device.tablet} {
+   //    order: 2;
+   //    width: 100%;
+   // }
```
Comment (jpoist97) [inline frontend/isoko/src/pages/Home.tsx:70]: Can you remove the comments from this file

**Review by jpoist97 — COMMENTED** [review]
Overall looks really good, don't have any comments that should need too much work to fix.

This is what it looks like for me. I know this task was just supposed to be the top half of the homepage but do you think you could just complete the homepage as a whole since we wanted to demo it this sprint and idk who else would pick it up this close to the end of the sprint. The only thing I mean by this is removing the signout button from the page and pushing the featured business stuff to the bottom.

Also can you fix up the styling discrepancies between this and the mockups like the black border on the search bar and also the slight gap between pictures 2 and 3 as well as on the left of the picture one and right of picture 3.

Screen Shot 2022-02-13 at 9 00 24 PM

File: frontend/isoko/src/pages/Home.tsx [hunk 4]
```diff
@@ -75,63 +74,59 @@ const RightImg = styled.div`
...
+            <LeftImg></LeftImg>
+            <MidImg></MidImg>
+            <RightImg></RightImg>
+         </ImageContainer>
+
+         <TopContentContianer>
+            <Title>ISOKO</Title>
+            <SubTitle>find something special</SubTitle>
+            <SearchBar />
+         </TopContentContianer>
```
Comment (shriyan44) [inline frontend/isoko/src/pages/Home.tsx:125]: correct container speling 121

**Review by ctperry0301 — DISMISSED** [review]
I won't review the code itself, as I believe JP did that, but the functionality works well. 

Make sure to fix ur Merge Conflicts, and Id suggest changing the leftmost picture, as it is pixelated and will need to be fixed eventually. Other than that it looks good to me.

File: frontend/isoko/src/components/Home/KeywordSearchBar.tsx [hunk 5]
```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+import Autocomplete from '@mui/material/Autocomplete';
+import TextField from '@mui/material/TextField';
+import keywordList from '../../constants/keywordList';
+
+// const options = ['Candy', 'Candle', 'Burger', 'Burger Hut'];
```
Comment (jpoist97) [inline frontend/isoko/src/components/Home/KeywordSearchBar.tsx:7]:
```suggestion
```

**Review by jpoist97 — APPROVED** [review]
Looks good, next time just merge your approved code in and make a new branch for your other task

**Review by jpoist97 — APPROVED** [review]

**Pull request merged**